includes/class-edd-logging.php

```diff
@@ -553,11 +553,8 @@ private function parse_args( $args = array() ) {
 
 		// Back-compat for post_parent.
 		if ( ! empty( $r['post_parent'] ) ) {
-			if ( ! empty( $r['log_type'] && 'file_download' === $r['log_type'] ) ) {
-				$r['product_id'] = $r['post_parent'];
-			} else {
-				$r['object_id'] = $r['post_parent'];
-			}
+			$type                                        = ! empty( $r['log_type'] ) ? $r['log_type'] : '';
+			$r[ $this->get_object_id_column_name_for_type( $type ) ] = $r['post_parent'];
```

Comment:
@robincornett we should probably unset the `post_parent` in the array `$r`, so it doesn't get passed in there, since this column doesn't exist.

Reply:
It is unset right after this in a group of old keys beginning in line 566.

Reply:
And so it is...thanks.